### DISCUSSION TITLE
Help: Atlas-based connectivity correlation matrix

### DISCUSSION THREAD

**Discussion by sunyunseung1** [HALFpipe/HALFpipe#544] [Q&A] [answered]
Hello,

I have a question regarding the 'correlation_matrix.txt' file generated as a result of the atlas-based connectivity calculation. Are the represented values r values or z values (Fisher's r-to-z transformed)?

Thank you in advance.

Best,
Sun

**Comment by HippocampusGirl** [comment, marked as answer]
Hi Sun, 
The `correlation_matrix.txt` contains the Pearson correlation, so the r-values.

Best,
Lea
